src/main/java/org/broadinstitute/hellbender/utils/codecs/gtf/EnsemblGtfCodec.java

```diff
@@ -29,12 +28,16 @@
     //==================================================================================================================
     // Private Static Members:
 
+    private static String VERSION_FIELD = "genome-version";
+    private static String DEFAULT_VERSION = "ENSEMBL_DEFAULT_VERSION";
+
     //==================================================================================================================
     // Private Members:
 
-    private final List<String> header         = new ArrayList<>();
-    private       int          currentLineNum = 1;
-    private       String       version        = "";
+    private final        List<String> header          = new ArrayList<>();
+    private              int          currentLineNum  = 1;
+    private              String       version         = null;
+    private static final Set<String>  commentPrefixes = Collections.unmodifiableSet(new LinkedHashSet<>(Arrays.asList("#!", "##")));
```

Comment:
COMMENT_PREFIXES and group it with the other statics

Reply:
rgr.  Fixed!